SharpLink/LavalinkManagerConfig.cs

```diff
@@ -5,10 +5,8 @@ namespace SharpLink
     public class LavalinkManagerConfig
     {
         // Default config follows the application.yml.example in the Lavalink GitHub located at https://github.com/Frederikam/Lavalink/blob/master/LavalinkServer/application.yml.example
-        public string WebSocketHost = "127.0.0.1";
-        public ushort WebSocketPort = 80;
-        public string RESTHost = "127.0.0.1";
-        public ushort RESTPort = 2333;
+        public string ServerAddress = "127.0.0.1";
+        public ushort ServerPort = 2333;
```

Comment:
In particular with this setup is you're now setting the host and port for both the rest and websocket system which *can* be changed in older versions of Lavalink. Breaking the support for older versions entirely. If they were to have different setups for the configuration.

Reply:
I'll revert these changes, but I think there needs to be a notice in the readme that states that the latest versions of Lavalink don't support two different host / ports. Devs will have to update their configs either way if they upgrade, and yeah at least it won't break backwards compatibility.

Reply:
On second thought, I think I'll make a compromise, I'll change the ports to be identical, but leave the support for backwards compatibility, so this should be okay? I think I'll still make a note in the readme that just clarifies why there's two different declarations for the server host and port.